rmgpy/data/thermo.py

```diff
@@ -1403,13 +1405,15 @@ def computeGroupAdditivityThermo(self, molecule):
                     raise
                 # Correct for gauche and 1,5- interactions
                 if not cyclic:
```

Comment:
Aren't your ortho/meta/para corrections going to never be used because these are always on cyclic?